### PR TITLE
	ticket 63, 31 - theming of social icons and qa header section and case study inner pages and footer changes

### PR DESCRIPTION
**Pull request by surbhi90**

### CODE REVIEW

File: sites/all/modules/features/adb_global_functions/adb_global_functions.module [hunk 1]
```diff
@@ -97,7 +97,7 @@ function adb_global_functions_block_view($delta = '') {
       // Social links.
       $links = array(
         array(
-          'title' => t('Facebook'),
+          'title' => t('<i class="fa fa-facebook"></i>'),
```
Comment (subhojit777) [inline sites/all/modules/features/adb_global_functions/adb_global_functions.module]: Wrong approach. No need to use `t()` or use string formatter https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/format_string/7

File: sites/all/modules/features/adb_qna/templates/adb-qna-header.tpl.php [hunk 2]
```diff
@@ -12,11 +12,11 @@
...
 ?>
 <div class="qna-header-wrapper">
-  <div class="type-name"><?php print $type_name; ?></div>
+  <div class="case-study-label"><?php print $type_name; ?></div>
   <div class="term-name-cards-wrapper">
-    <div class="term-name"><?php print $current_term_name; ?></div>
+    <h1 class="detail-pages-heading"><?php print $current_term_name; ?></h1>
     <div class="cards-wrapper">
-      <div class="cards-toggler"><?php print t('Change cards'); ?></div>
+      <div class="cards-toggler"><?php print t('Change cards <i class="fa fa-angle-down"></i>'); ?></div>
```
Comment (subhojit777) [inline sites/all/modules/features/adb_qna/templates/adb-qna-header.tpl.php]: Wrong approach.

Comment (surbhi90) [inline sites/all/modules/features/adb_qna/templates/adb-qna-header.tpl.php]: Use this:
```
t('Change cards !icon', array(
'!icon' => '<i class="fa fa-angle-down"></i>',
));
```

File: sites/all/themes/custom/adb/sass/pages/_caseStudy.scss [hunk 3]
```diff
@@ -408,6 +442,31 @@
...
+  .case-study-tags {
+    margin-bottom: 0;
+    li {
+      background-color: transparent;
+      border: none;
+    }
+  }
+  .item-list {
+    ul li {
+  	  margin: 0 16px 8px 0;
```
Comment (subhojit777) [inline sites/all/themes/custom/adb/sass/pages/_caseStudy.scss]: correct this indentation.

**Pull request merged**